runtime-assets.sln

```diff
@@ -3,6 +3,8 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
 VisualStudioVersion = 15.0.26124.0
 MinimumVisualStudioVersion = 15.0.26124.0
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Private.Runtime.UnicodeData", "src\Microsoft.Private.Runtime.UnicodeData\Microsoft.Private.Runtime.UnicodeData.csproj", "{6E7ACB67-F024-4289-A86E-482A88BEE7D5}"
```

Comment:
I'd prefer `System.Private.Runtime.UnicodeData` for the name, unless there's some precedent for having the `Microsoft.Private` convention.